packages/app/src/components/home/HomePage.tsx

```diff
@@ -55,7 +55,7 @@ export const HomePage = () => (
     <Content>
       <Grid container spacing={3}>
         <Grid item xs={12}>
-          <HomePageSearchBar />
+          <HomePageSearchBar debounceTime={100} />
```

Comment:
Hm. So this one is getting different debounce values in different contexts - is that important? Just checking if maybe we can revert this and the other place where it's set, and change to a sensible default like perhaps 200 (not zero!) for it in the class. That would also mean that users get this sensible behavior automatically, without us having to change the create-app template and writing instructions for users to make modifications in their own app code accordingly.

Reply:
We leave that decision up to the users, they can decide if it's important or not to use debounce, and if so, they can customize the debounce time to a value that works best for them. It's an optional property, I removed it from the example app

Reply:
I like the idea of having it configurable but set the default value to 200 or something as @freben suggest!

Reply:
What do you think? @camilaibs